javascript/code-challenges/challenges-10.test.js

```diff
@@ -0,0 +1,364 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named returnTen, takes in a string and uses split and splice to return the last 10 characters from that string as elements of an array.
+
+------------------------------------------------------------------------------------------------ */
+
+function returnTen(str){
+  return str.split('').splice(str.length - 10, 10);
+}
```

Comment:
This is funny. Did the challenge come like this, using function syntax instead of arrow functions?